CHANGELOG.md

```diff
@@ -1,5 +1,46 @@
 # 📜 BayesFilters changelog
 
+## Version 0.9.101
```

Comment:
All the entries of this chapter must be muved in chapter `Version 0.8.101`.
Moreover, all the entries of `Version 0.7.101` must also be moved to `Version 0.8.101` because we are making breaking changes.
Finally, since we are moving to version `0.8`, devel version should just be `0.8.100`, i.e. we are not developing toward release version `0.8.1`.

Reply:
Ok, now is way more clear. Thank you for the explanation.

Reply:
Here for that 👍

Reply:
[This](https://semver.org/) is the official page of SemVer. Have a look at it 😉 
For this project, I opted for a slightly different version detailed [here](https://github.com/robotology/bayes-filters-lib#%EF%B8%8F-about-versioning).

Reply:
I made the changes to CHANGELOG.md and tried to "merge" things using the right categories.

However it seems that some classes/methods that were referred in the 0.7.101 do not exist anymore or changed their name:
- class `Process`
- class `GenericData` (this should be what now is called `Data`)
- typedefs `Eigen*Data`
- method `MeasurementModel::registerProcessData`
- method `MeasurementModel::getProcessMeasurements`

Do you think I can remove these things from the CHANGELOG?

Reply:
Also, if something was created and then removed, like the class 'Particle' can I change things like

```
- Added Particle and ParticleSet classes.
- Removed Particle class
```
to 
```
- Added ParticleSet classes.
```

Reply:
Yes absolutely! 👍 
Just make the CHANGELOG consistent with the status of the project up to the very last commit 😃

Reply:
Perfect :+1: